logger_test.go

```diff
@@ -57,7 +94,7 @@ func TestLogWithFieldsAndStacktrace(t *testing.T) {
 	// test stack trace strings
 	assert.Contains(t, buffer.String(), "github.com/uniplaces/go-logger.justToShowUpInStackTrace")
 	assert.Contains(t, buffer.String(), "github.com/uniplaces/go-logger.TestLogWithFieldsAndStacktrace")
-	assert.Contains(t, buffer.String(), "github.com/uniplaces/go-logger/logger_test.go:84")
+	assert.Contains(t, buffer.String(), "github.com/uniplaces/go-logger/logger_test.go:122")
```

Comment:
can we make this test deterministic? so it doesn't break with every change to this file

Reply:
Move the test method to the top of the file?

Reply:
maybe use a regex to check this condition: `(github.com\/uniplaces\/go-logger\/logger_test.go):[0-9]+` seems to work, and you can use assert's function `Regexp`

Reply:
The intention it to test where in the file the error was triggered.
This change would make it so we would only test the error happened in the test file.